modules/comment-likes/admin-style.css

```diff
@@ -0,0 +1,55 @@
+.vers img {
+    display: none;
+}
+
+.metabox-prefs .vers img {
+    display: inline;
+}
+
+.fixed .column-comment_likes {
+    width: 5.5em;
+    padding: 8px 0;
+    text-align: center;
+}
+
+.fixed .column-stats {
+    width: 5em;
+}
+
+.fixed .column-comment_likes .comment-like-count {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    display: inline-block;
+    padding: 0 8px;
+    height: 2em;
+    margin-top: 5px;
+    -webkit-border-radius: 5px;
+    border-radius: 5px;
+    background-color: #72777C;
+    color: #FFF;
+    font-size: 11px;
+    line-height: 21px;
+}
+
+.fixed .column-comment_likes .comment-like-count:after {
+    border: none !important;
```

Comment:
How strictly necessary is the `!important` here?  I know there's people who give us crap every time we use it, so if we can at least cite an instance here where it's necessary, it would be appreciated.